### PR TITLE
fix: Construction of OIDC endpoint when rootPath has a trailing slash

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

Part of https://github.com/stackabletech/operator-rs/pull/910

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] CRD documentation for all fields, following the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs/style-guide).
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
- [ ] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] Code contains useful logging statements
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs/style-guide).
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -30,3 +30,4 @@ tracing = "0.1"
 
 # [patch."https://github.com/stackabletech/operator-rs.git"]
 # stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "main" }
+# stackable-operator = { path = "../operator-rs/crates/stackable-operator" }
```
[the remark this answers is not in the document]
Comment (sbernauer) [reply Cargo.toml:33]: Yep, I added it intentional, as it saves time during development when you quickly want to test a local operator-rs version.

**Review by labrenbe — APPROVED** [review]
lgtm

**Review by labrenbe — APPROVED** [review]

**Comment by sbernauer** [comment]
Started https://testing.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/airflow-operator-it-custom/13/

**Comment by sbernauer** [comment]
The test takes >2h, but the relevant OIDC tests already passed

**Comment by lfrancke** [comment]
Did this make it into 24.11?

**Comment by sbernauer** [comment]
Nope, but it will be part of 24.11.1. Not sure what label we should use here ^^

**Pull request merged**